frontend/components/site/SiteSettings/BasicSiteSettings.jsx

```diff
@@ -67,8 +74,9 @@ export const BasicSiteSettings = ({
           <div className="well">
             <h3 className="well-heading">Demo Site</h3>
             <p className="well-text">
-              Set a branch to be deployed to a custom demo URL like demo.yoursite.gov
-              instead of to a standard Federalist preview URL.
+              Optional: After setting up DNS with the Federalist team, set a demo branch
```

Comment:
Moved the word "Optional" to the start of this text instead of having it at the end, which looked like it was dangling to me.

Reply:
agree